### DISCUSSION TITLE
New unit requests: %w/w, %w/v, %v/v

### DISCUSSION THREAD

**Discussion by meganrwong** [qudt/qudt-public-repo#1451] [General]
w - weight
v - volume 
Concentration units used in chemistry. Appropriate to add? 
example here http://purl.obolibrary.org/obo/UO_0000163

**Comment by steveraysteveray** [comment]
Yes, these should be additional units related to the quantity kind:

http://qudt.org/vocab/quantitykind/VolumeFraction for the percent volume fraction unit,
and we would need to define new quantity kinds for 
percent weight fraction unit
and
percent weight density unit, or something like that...
